pkg/cidata/cidata.TEMPLATE.d/boot/05-lima-mounts.sh

```diff
@@ -14,7 +14,16 @@ if [ -d /sys/fs/selinux ]; then
 	for line in $(grep -n virtiofs </etc/fstab | cut -d':' -f1); do
 		OPTIONS=$(awk -v line="$line" 'NR==line {print $4}' /etc/fstab)
 		if [[ ${OPTIONS} != *"context"* ]]; then
-			sed -i -e "$line""s/comment=cloudconfig/comment=cloudconfig,context=\"system_u:object_r:container_file_t:s0\"/g" /etc/fstab
+			##########################################################################################
+			## When using vz & virtiofs, initially container_file_t selinux label
```

Comment:
indentation seems broken

Reply:
Lint looks ok 🤔

Reply:
Sorry, looks like just a glitch of github web UI